### PR TITLE
ENH tests for priors, removed some, bug fixes, documentation

### PR DESCRIPTION
**Pull request by beckermr**
This PR continues writing tests for priors.

### CODE REVIEW

**Comment by esheldon** [comment]
What's your goal for this PR?  What kind of tests do you have in mind?

**Comment by beckermr** [comment]
See the rest of the tests I already merged. They do things like make sure the draws work correctly with the right outputs, make sure the fitting works where needed, etc.

File: ngmix/priors/priors.py [hunk 1]
```diff
@@ -1458,6 +1503,12 @@ def fit(self, x, y):
         return res
 
 
+############################################
+# MRB: I've gone through the first 1k lines.
+# I am going to stop and write tests, then split this module into pieces
+# and then do docs for the rest.
+
+
```
Comment (beckermr) [inline ngmix/priors/priors.py:1511]: We should remove this comment if we have tests for the rest.

**Review by beckermr — COMMENTED** [review]
I want to go through and check the doc strings of anything we wrote a test for in this PR before we merge it.

**Comment by esheldon** [comment]
ready to go?

**Comment by beckermr** [comment]
Nope. I want to go through the doc strings.

**Comment by beckermr** [comment]
ok @esheldon, I am going to run a pixel level test of this code branch against the previous one. If that passes, we can merge.

**Pull request merged**